### DISCUSSION TITLE
cjs-esm-exports deprecation

### DISCUSSION THREAD

**Discussion by arcanis** [esm-dev/esm.sh#500] [Q&A] [answered]
We use `cjs-esm-exports` in an internal project, but the [npm page](https://www.npmjs.com/package/cjs-esm-exports) reports it as deprecated. Is it the case? What's the recommended replacement?

**Comment by ije** [comment, marked as answer]
i renamed it to `esm-cjs-lexer`, check https://www.npmjs.com/package/esm-cjs-lexer
